lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt

```diff
@@ -366,6 +368,51 @@ class CheckableChipView @JvmOverloads constructor(
         touchFeedbackDrawable.setHotspot(x, y)
     }
 
+    override fun onSaveInstanceState(): Parcelable {
+        return SavedState(super.onSaveInstanceState()).apply {
+            checked = isChecked
+        }
+    }
+
+    override fun onRestoreInstanceState(state: Parcelable?) {
+        val savedState = state as SavedState
+        super.onRestoreInstanceState(savedState.superState)
+
+        val listener = onCheckedChangeListener
```

Comment:
@rom4ek I'm not sure if there's a better way to do this. It basically prevents the listener to be invoked while the state is still restored.

Reply:
I think it's fine, cause there are some other ways to do so, but the implementation of them is similar to your's one, so I'd leave it like this probably.